llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.h

```diff
@@ -33,6 +37,7 @@ enum class DeviceLibExt : std::uint32_t {
 };
 
 using SYCLDeviceLibFuncMap = std::unordered_map<std::string, DeviceLibExt>;
+
 class SYCLDeviceLibReqMaskPass : public ModulePass {
```

Comment:
It's not a to-do comment.
@jinge90 @AlexeySachkov I'm just wondering why this scanning was implemented as a pass?
Might it be simpler to define it as a helper function?

Reply:
> @jinge90 @AlexeySachkov I'm just wondering why this scanning was implemented as a pass?

I'm honestly don't know the history of this pass.

> Might it be simpler to define it as a helper function?

That makes sense to me, we don't really need a pass here, but rather a separate function, which produces a certain result from an input LLVM IR module.

Reply:
Submitted an issue https://github.com/intel/llvm/issues/5362.